src/histogram/dtos/stars.ts

```diff
@@ -0,0 +1,31 @@
+import { ApiProperty, ApiPropertyOptional } from "@nestjs/swagger";
+import { Type } from "class-transformer";
+import { IsEnum, IsIn, IsInt, IsOptional, IsString } from "class-validator";
+import { OrderDirectionEnum } from "../../common/constants/order-direction.constant";
+
+export class StarsHistogramDto {
+  @ApiPropertyOptional({
+    description: "Range in days",
+    default: 30,
+    type: "integer",
+  })
+  @Type(() => Number)
+  @IsIn([7, 30, 90])
+  @IsInt()
+  @IsOptional()
+  readonly range?: number = 30;
+
+  @ApiProperty({
+    description: "Repo name",
+    type: "string",
+    example: "open-sauced/app",
+  })
+  @Type(() => String)
+  @IsString()
+  repo_name: string;
```

Comment:
Only for consistency, but not a blocker

```suggestion
  repo: string;
```